### ISSUE TITLE
Fortune sha256 mismatch for el_capitan

### ISSUE THREAD

**Issue by mkaag** [Homebrew/homebrew#45888]
From a fresh OSX El Capitan, I ran the cmd **brew install fortune** and got an sha256 error:

```
==> Downloading https://homebrew.bintray.com/bottles/fortune-9708.el_capitan.bottle.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "fortune"
Download failed: https://homebrew.bintray.com/bottles/fortune-9708.el_capitan.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading http://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz
Already downloaded: /Library/Caches/Homebrew/fortune-9708.tar.gz
Error: SHA256 mismatch
Expected: 1a98a6fd42ef23c8aec9e4a368afb40b6b0ddfb67b5b383ad82a7b78d8e0602a
Actual: b288b71f393c126f6897a65f722328dceb9c67ff9be9bf0d3e576f2ed91c4ef6
Archive: /Library/Caches/Homebrew/fortune-9708.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by DomT4** [comment]
It's okay here:
```
==> Downloading http://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz
/usr/bin/curl -fLA Homebrew 0.9.5 (Ruby 2.0.0-645; OS X 10.11.2) http://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz -C 0 -o /usr/local/var/homebrew/cache/fortune-9708.tar.gz.incomplete
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1375k  100 1375k    0     0   387k      0  0:00:03  0:00:03 --:--:--  387k
Downloaded to: /usr/local/var/homebrew/cache/fortune-9708.tar.gz
SHA1: 9f96a00dbf5d2c6b0e7324af0eae988ce6adc5ad
SHA256: 1a98a6fd42ef23c8aec9e4a368afb40b6b0ddfb67b5b383ad82a7b78d8e0602a
==> Verifying fortune-9708.tar.gz checksum
```

Does it reproduce consistently if you `brew fetch fortune -s -f`?

**Comment by mkaag** [comment]
Hi Dom,

running `brew fetch fortune -s -f` produces the following:

```
==> Downloading http://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz
######################################################################## 100.0%
Downloaded to: /Library/Caches/Homebrew/fortune-9708.tar.gz
SHA1: e9c164677e414c3b649f12185e01f4c7ea2f083c
SHA256: b288b71f393c126f6897a65f722328dceb9c67ff9be9bf0d3e576f2ed91c4ef6
Warning: Formula reports different SHA256: 1a98a6fd42ef23c8aec9e4a368afb40b6b0ddfb67b5b383ad82a7b78d8e0602a
```

**Comment by DomT4** [comment]
Strange. Does `brew doctor` warn about anything interesting?

**Comment by mkaag** [comment]
Good question!

Only warning related to unbrewed header files in `/usr/local/include/osxfuse_lima/fuse` (which was installed after trying to install fortune) and keg-only formula related to openssl.

**Comment by DomT4** [comment]
Nothing there that should be problematic :confused:.

If you open `http://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz` in your browser of choice, let it download and then do `shasum -a 256 ~/Downloads/fortune-mod-9708.tar.gz` (presuming the file downloads to Downloads) what checksum does it push out?

**Comment by mkaag** [comment]
No better result using the browser, I give a try with `wget` and got the following:

```
$ wget ftp://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz
--2015-11-17 11:16:53--  ftp://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz
           => 'fortune-mod-9708.tar.gz'
Resolving ftp.ibiblio.org... 152.19.134.40
Connecting to ftp.ibiblio.org|152.19.134.40|:21... connected.
Logging in as anonymous ... Logged in!
==> SYST ... done.    ==> PWD ... done.
==> TYPE I ... done.  ==> CWD (1) /pub/linux/games/amusements/fortune ... done.
==> SIZE fortune-mod-9708.tar.gz ... 1408517
==> PASV ... done.    ==> RETR fortune-mod-9708.tar.gz ...

Error in server response, closing control connection.
Retrying.
```

Looks like  I do have an issue connecting to the server, do you experience the same behaviour?

**Comment by DomT4** [comment]
Very oddly, I can do `brew fetch fortune` and it works and there are the expected files in the tarball, but I can't manually retrieve it via either `wget` or `curl`. Just hangs.

**Comment by rakkesh** [comment]
@mkaag use `http` instead of `ftp`. The formula has `http` url.

```
$ wget http://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz                                                                                                                                   
--2015-11-21 19:22:33--  http://ftp.ibiblio.org/pub/linux/games/amusements/fortune/fortune-mod-9708.tar.gz
Resolving ftp.ibiblio.org... 152.19.134.40
Connecting to ftp.ibiblio.org|152.19.134.40|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1408517 (1.3M) [application/x-gzip]
Saving to: 'fortune-mod-9708.tar.gz'

fortune-mod-9708.tar.gz                                  100%[=================================================================================================================================>]   1.34M   230KB/s   in 7.4s   

2015-11-21 19:22:41 (186 KB/s) - 'fortune-mod-9708.tar.gz' saved [1408517/1408517]
```

By the way, `El Capitan` bottle is missing[404] for `fortune`. It's worth noting this in https://github.com/Homebrew/homebrew/issues/46168.

**Comment by bfontaine** [comment]
@rakkesh Thanks; I updated the issue.

**Comment by mikemcquaid** [comment]
This should be fixed as-of c07e70a. Can you `brew update` and try again?

**Comment by mkaag** [comment]
I confirm the pb is fixed after `brew update`, tx folks!
